### ISSUE TITLE
Ranking - A screen reader doesn't announce a question's title when a question receives focus and a ranking item is selected

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#10920]
T26464 - Ranking - A screen reader doesn't announce a question's title when a question receives focus and a ranking item is selected
https://surveyjs.answerdesk.io/internal/ticket/details/T26464
***
When a Radiogroup is focused, it's title is announced. However, when a Ranking receives focus, no question title is announced: [View Demo](https://plnkr.co/edit/F8R4LWICVpaTMBFy).

<img width="1382" height="966" alt="Image" src="https://github.com/user-attachments/assets/53d03007-517f-4f07-92e8-0b90f9fd2a2a" />
